src/biosystems/wellness/cache.py

```
@@ -0,0 +1,616 @@
+
+    if avg_stress is not None:
+        if avg_stress > str_red:
+            red_signals.append(f"Stress {avg_stress:.0f}")
+        elif avg_stress > str_amber:
+            amber_signals.append(f"Stress {avg_stress:.0f}")
+
+    if resp_rate is not None and rr_thresholds:
+        rr_red   = rr_thresholds.get("red")
+        rr_amber = rr_thresholds.get("amber")
+        if rr_red and resp_rate >= rr_red:
+            red_signals.append(f"Resp rate {resp_rate:.1f} brpm (+{rr_sigma:.1f}σ)")
```

Ruff (0.15.6)

[warning] String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)